web/packages/teleport/src/services/storageService/types.ts

```diff
@@ -36,6 +36,7 @@ export const KeysEnum = {
   USERS_NOT_EQUAL_TO_MAU_ACKNOWLEDGED:
     'grv_users_not_equal_to_mau_acknowledged',
   LOCAL_NOTIFICATION_STATES: 'grv_teleport_notification_states',
+  RECENT_HISTORY: 'grv_teleport_recent_history',
```

Comment:
should this mention "sidenav" just so it isn't broad enough  like `grv_teleport_sidenav_recent_history` ?